setup.py

```diff
@@ -69,9 +68,18 @@ def launch_http_server(directory):
 if command not in ['test', 'coverage']:
     long_description = open('README.rst').read()
 
+install_requires = []
+if sys.version_info < (2, 7):
+    install_requires.extend([
+        'argparse',
+        'ordereddict',
+        'unittest2',
+    ])
+
+
 setup(
     name='ConfigArgParse',
-    version=configargparse.__version__,
```

Comment:
You could read the `__init__` file, like I do in https://github.com/jayvdb/travis_log_fetch/blob/7c3e6925f77579d3d4e0c73d8e8ccb0151714150/setup.py#L57